lib/components/user/trip-basics-pane.js

```diff
@@ -31,73 +37,142 @@ const StyledLabel = styled.label`
 `
 
 const allDays = [
-  { name: 'monday', text: 'Mon.' },
-  { name: 'tuesday', text: 'Tue.' },
-  { name: 'wednesday', text: 'Wed.' },
-  { name: 'thursday', text: 'Thu.' },
-  { name: 'friday', text: 'Fri.' },
-  { name: 'saturday', text: 'Sat.' },
-  { name: 'sunday', text: 'Sun.' }
+  { name: 'monday', text: 'Mon.', fullText: 'Mondays' },
+  { name: 'tuesday', text: 'Tue.', fullText: 'Tuesdays' },
+  { name: 'wednesday', text: 'Wed.', fullText: 'Wednesdays' },
+  { name: 'thursday', text: 'Thu.', fullText: 'Thursdays' },
+  { name: 'friday', text: 'Fri.', fullText: 'Fridays' },
+  { name: 'saturday', text: 'Sat.', fullText: 'Saturdays' },
+  { name: 'sunday', text: 'Sun.', fullText: 'Sundays' }
 ]
 
+/**
+ * @returns true if there is a trip matching for the specified availability/existence check.
+ */
+function itineraryExists (dayCheck) {
+  return dayCheck && dayCheck.valid
```

Comment:
I'm not sure this method is needed any longer, is it? We should be guaranteed to have an entry for each day in the ItineraryExistence response, right?

Reply:
No it is not. Updated in https://github.com/opentripplanner/otp-react-redux/pull/242/commits/846892ab816dc32ccd6688b06042f030225c453a.